mono/metadata/console-unix.c

```diff
@@ -254,7 +254,8 @@ do_console_cancel_event (void)
 	method = mono_class_get_method_from_name (klass, "BeginInvoke", -1);
 	g_assert (method != NULL);
 
-	mono_threadpool_ms_begin_invoke (domain, (MonoObject*) load_value, method, NULL);
+	mono_threadpool_ms_begin_invoke (domain, (MonoObject*) load_value, method, NULL, &error);
+	mono_error_cleanup (&error);
```

Comment:
What am I allowed to do on the finalizer thread? I can't raise here, can I? The best I can do is `g_warning (... , mono_error_get_message(&error))`, I think?